.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,27 +4,34 @@ about: Create a report to help us improve
 title: ''
 labels: bug
 assignees: ''
-
 ---
 
 **Describe the bug**
+
 A clear and concise description of what the bug is.
 
 **To Reproduce**
+
 Code or steps to reproduce the error
+
 ```python
 # Put your code here
 ```
 
 **Error traceback**
+
 If applicable please provide full traceback of the error.
+
 ```
 # Put traceback here
 ```
 
 **Expected behavior**
+
 A clear and concise description of what you expected to happen.
 
 **Desktop (please complete the following information):**
- - probatus version [e.g. 1.5.0]
- - OS: [e.g. iOS]
+
+ - probatus version [e.g. 1.5.0] (`pip list | grep probatus`)
```

Comment:
Small Suggestion: Maybe the Desktop description should be higher, in case someone skips some sections. Even before Error traceback. This one is really important.

What do you think?

Reply:
done.